endpoint/opcua/src/main/java/de/fraunhofer/iosb/ilt/faaast/service/endpoint/opcua/listener/AasServiceMethodManagerListener.java

```diff
@@ -60,32 +57,15 @@ public AasServiceMethodManagerListener(OpcUaEndpoint ep, AasServiceNodeManager n
     }
 
 
-    /**
-     * Callback method when a method was called
-     *
-     * @param serviceContext the current service context
-     * @param objectId the ID of the node whose method is being called
-     * @param object the object node whose method is being called, if available
-     * @param methodId the ID of the method being called
-     * @param method the method node being called, if available
-     * @param inputArguments input argument values
-     * @param inputArgumentResults argument errors. If errors in the values are encountered.
-     * @param inputArgumentDiagnosticInfos diagnostic info, in case of errors.
-     * @param outputs output values. The array is pre-created, just fill in the values.
-     * @return true if you handle the call, which prevents any other handler being called.
-     * @throws StatusException if there are errors in the method handling. For example, if you set inputArgumentResults,
-     *             you should throw a StatusException with StatusCodes.Bad_InvalidArgument
-     */
     @Override
     public boolean onCall(ServiceContext serviceContext, NodeId objectId, UaNode object, NodeId methodId, UaMethod method, Variant[] inputArguments,
                           StatusCode[] inputArgumentResults, DiagnosticInfo[] inputArgumentDiagnosticInfos, Variant[] outputs)
             throws StatusException {
 
         boolean retval = false;
 
-        // Handle method calls
         // Note that the outputs array is already allocated
-        LOGGER.info("onCall: method {}: called. InputArguments: {}", methodId, inputArguments);
+        LOGGER.trace("onCall: method {}: called. InputArguments: {}", methodId, inputArguments);
 
         try {
             if (endpoint == null) {
```

Comment:
Move null-check of endpoint to constructor

Reply:
moved to constructor